homeassistant/components/homekit_controller/sensor.py

```diff
@@ -97,6 +97,13 @@ class HomeKitSensorEntityDescription(SensorEntityDescription):
         state_class=SensorStateClass.MEASUREMENT,
         native_unit_of_measurement=PRESSURE_HPA,
     ),
+    CharacteristicsTypes.Vendor.VOCOLINC_OUTLET_ENERGY: HomeKitSensorEntityDescription(
+        key=CharacteristicsTypes.Vendor.VOCOLINC_OUTLET_ENERGY,
+        name="Real Time Energy",
```

Comment:
It's a bit weird to call this energy when it measures in watts and has device class power.

Reply:
There are a few examples of this in this file and it traces back to a vendor calling it that. Happy to put in a PR to change the default names for them all if you have an alternative in mind.

Reply:
If it's a power measurement I'd call it something with power.